handlers/server_test.go

```diff
@@ -0,0 +1,615 @@
+
+			newEmployee := store.Employee{
+				Name:       "shayan",
+				Email:      "shayan@twilio.com",
+				EmployeeId: "1",
+				City:       "Yar",
+				Address:    "The Black Pearl",
+				Department: "Piratology",
+			}
+
+			item, _ := dynamodbattribute.MarshalMap(newEmployee)
+
+			m.GetError = nil
+			m.GetOutput = dynamodb.GetItemOutput{
+				Item: item,
+			}
+
+			server.GetEmployee(w, req)
+
+			res := w.Result()
+			defer res.Body.Close()
+
+			var resp GetEmployeeResponse
+
+			err = json.NewDecoder(res.Body).Decode(&resp)
+			if err != nil {
+				fmt.Printf("unable to successfully decode response got error %v", err)
+			}
+
+			Expect(resp.Status).To(Equal(http.StatusOK))
+			Expect(resp.Data).To(Equal(newEmployee))
+		})
+		It("Status:201 Post Employee Success", func() {
+			reader := strings.NewReader(`{"employeeName": "pablo", "city":  "Denver", "address": "3 Main St", "department": "eng"}`)
+
+			req, err := http.NewRequest("POST", "/v1/employee/add", reader)
+			if err != nil {
+				fmt.Printf("could not create a request: %v", err.Error())
+			}
+
+			w := httptest.NewRecorder()
+
+			server.PostEmployee(w, req)
+
+			res := w.Result()
+			defer res.Body.Close()
+
+			var resp GetOneEmployee
+
+			err = json.NewDecoder(res.Body).Decode(&resp)
+			if err != nil {
+				fmt.Printf("unable to successfully decode response got error %v", err)
+			}
+
+			Expect(resp.Status).To(Equal(http.StatusCreated))
+			Expect(resp.Employee.Name).To(Equal("pablo"))
+			Expect(resp.Employee.Email).To(Equal("pablo@twilio.com"))
+			//UUID REGexp
+			r := regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$")
+			val := r.MatchString(resp.Employee.EmployeeId)
+			Expect(val).To(Equal(true))
+			Expect(resp.Employee.City).To(Equal("Denver"))
+			Expect(resp.Employee.Address).To(Equal("3 Main St"))
+			Expect(resp.Employee.Department).To(Equal("eng"))
+		})
+		It("Status:204 Delete Employee Success", func() {
+			//some UUID
+			uuid := uuid.New().String()
+			reader := strings.NewReader(fmt.Sprintf(`{"employeeId": "%v"}`, uuid))
+
+			req, err := http.NewRequest("DELETE", "/v1/employee/delete", reader)
+			if err != nil {
+				fmt.Printf("could not create a request: %v", err.Error())
+			}
+
+			w := httptest.NewRecorder()
+
+			server.DeleteEmployee(w, req)
+
+			res := w.Result()
+			defer res.Body.Close()
+
+			var resp DeletionResponse
+
+			err = json.NewDecoder(res.Body).Decode(&resp)
+			if err != nil {
+				fmt.Printf("unable to successfully decode response got error %v", err)
+			}
+
+			Expect(resp.Status).To(Equal(http.StatusNoContent))
+			Expect(resp.Msg).To(Equal("Successful Deletion of Employee"))
+		})
+		It("Status 200: Update Employee Success", func() {
+			reader := strings.NewReader(`{"employeeName": "pablo", "email":"pablo@yahoo.com", "city":  "Denver", "address": "3 Main St", "department": "eng"}`)
+
+			req, err := http.NewRequest("PATCH", "/v1/employee/{id}/update", reader)
```

Comment:
```suggestion
			req, err := http.NewRequest(http.MethodPatch, "/v1/employee/{id}/update", reader)
```

we want to use the stdlib's constants for this kind of thing whenever possible